### ISSUE TITLE
Add detailed Readme file

### ISSUE THREAD

**Issue by itsmeabhishek03** [ItIsOHM/gfi-notifier#3]
Hey @ItIsOHM i can add a detailed Readme file for this project which will help participants to run the project on thier localhost. Please assign this to me under SWOC

**Comment by ItIsOHM** [comment]
Okay @itsmeabhishek03 I'm assigning this to you, let me know if you want any more information.

**Comment by itsmeabhishek03** [comment]
hey @ItIsOHM in backend you have lot of .env variables for eg -

   const DB_URL = process.env.DB_URI;

my query is how participants will get this variables, as you have not uploaded the .env file on github
it is required becuse without it backend won't run and i wanted to add this in README.md file as you want a detail readme.

**Comment by govind-kumarr** [comment]
> hey @ItIsOHM in backend you have lot of .env variables for eg -
> 
> const DB_URL = process.env.DB_URI;
> 
> my query is how participants will get this variables, as you have not uploaded the .env file on github it is required becuse without it backend won't run and i wanted to add this in README.md file as you want a detail readme.

Environment variables are secret and can change according to project setup. They can't be uploaded to github or any other platform.

**Comment by itsmeabhishek03** [comment]
> > hey @ItIsOHM in backend you have lot of .env variables for eg -
> > const DB_URL = process.env.DB_URI;
> > my query is how participants will get this variables, as you have not uploaded the .env file on github it is required becuse without it backend won't run and i wanted to add this in README.md file as you want a detail readme.
> 
> Environment variables are secret and can change according to project setup. They can't be uploaded to github or any other platform.

Yes, I know they are secret key that's why I asked you. Suppose, what if someone want to work on backend part how will they test it until they will be not able to run it ?

**Comment by itsmeabhishek03** [comment]
> > > hey @ItIsOHM in backend you have lot of .env variables for eg -
> > > const DB_URL = process.env.DB_URI;
> > > my query is how participants will get this variables, as you have not uploaded the .env file on github it is required becuse without it backend won't run and i wanted to add this in README.md file as you want a detail readme.
> > 
> > 
> > Environment variables are secret and can change according to project setup. They can't be uploaded to github or any other platform.
> 
> Yes, I know they are secret key that's why I asked you. Suppose, what if someone want to work on backend part how will they test it until they will be not able to run it ?

**Comment by TahaHameed23** [comment]
For the dev part, you should generate the keys yourself, and the repo owner should provide:
1.  .env.example (so that contributors can know which keys to generate)
2.  mongoose model/schema

**Comment by itsmeabhishek03** [comment]
> For the dev part, you should generate the keys yourself, and the repo owner should provide:
> 
> 1. .env.example (so that contributors can know which keys to generate)
> 2. mongoose model/schema

 @TahaHameed23 Exactly bro, that's why I am asking about mongoose key or url

**Comment by yaten2302** [comment]
Hi @ItIsOHM, I've created a PR for adding the `.env.example` file, could you please review and merge it under SWOC

**Comment by ItIsOHM** [comment]
@itsmeabhishek03 sorry for the late reply. I have merged a PR by @yaten2302 that makes a `.env.example` file. Let me know if you want further help :)

**Comment by yaten2302** [comment]
@ItIsOHM, thanks for merging my PR, could you also assign this issue to me and also give the level label so that it can be counted in SWOC.

**Comment by ItIsOHM** [comment]
@yaten2302 currently @itsmeabhishek03 is assigned on this issue so I can't assign you yet. And thanks for the label mentions, will add those :)
